hibernate-core/src/main/java/org/hibernate/hql/spi/id/inline/AbstractInlineIdsBulkIdHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.hql.spi.id.inline;
+
+import java.util.List;
+
+import org.hibernate.engine.spi.QueryParameters;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.hql.internal.ast.HqlSqlWalker;
+import org.hibernate.hql.spi.id.AbstractIdsBulkIdHandler;
+
+/**
+ * Bulk-id strategy for all the strategies that inline the identifiers of the updatable/deletable rows.
```

Comment:
Maybe this comment could be updated - in my mind it's not a bulk-id strategy, but rather a base class for handlers.